x-pack/legacy/plugins/canvas/canvas_plugin_src/functions/common/lt.ts

```diff
@@ -37,7 +37,6 @@ export function lt(): ExpressionFunction<'lt', Context, Arguments, boolean> {
         return false;
       }
 
-      // @ts-ignore #35433 This is a wonky comparison for nulls
```

Comment:
Why was this removed?

https://github.com/elastic/kibana/issues/35433

Reply:
Ah I fixed this in #37305, but I didn't realize there was an issue to close. My fault for skimming over the comment. I removed `boolean`  and `null` as valid `context` types for `gt`, `gte`, `lt`, and `lte`. The comment was left behind, so I'm deleting it here, but the change to the function def was merged already.